cfgov/unprocessed/js/modules/util/web-storage-proxy.js

```diff
@@ -81,6 +81,7 @@ function setStorage( storage ) {
   _storage = storage;
 }
 
+/* istanbul ignore next */
```

Comment:
It seems like that ignore next is redundant since it's still counting it towards coverage somehow. 
![image](https://cloud.githubusercontent.com/assets/1860176/13931609/818d1be8-ef7a-11e5-9954-0ad08b593ef0.png)